doc/example/exampleBatchFile.slurm

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+## Give your job a name to distinguish it from other jobs you run.
+#SBATCH --job-name=Test
+#SBATCH --partition=short
+## use --exclusive to get the whole nodes exclusively for this job
+#SBATCH --exclusive
+
+## Separate output and error messages into 2 files.
+## NOTE: %u=userID, %x=jobName, %N=nodeID, %j=jobID, %A=arrayID, %a=arrayTaskID
+#SBATCH --output=logBonna/%x-%j-%j.log
+## Specify how much memory your job needs. (2G is the default)
+#SBATCH --mem-per-cpu=5800        # Total memory needed per task (units: K,M,G,T)
+
+## Specify how much time your job needs. (default: see partition above)
+#SBATCH --time=0-08:00:00   # Total time needed for job: Days-Hours:Minutes
+
+#SBATCH --ntasks=3   # 2 workers, 1 manager
+
+## Load the relevant modules needed for the job
+module purge
+module load OpenMPI/4.0.1-GCC-8.3.0-2.32
+module load Python/3.8.2-GCCcore-9.3.0
+module load SWIG/4.0.1-GCCcore-8.3.0 HDF5/1.10.6-gompi-2020a OpenBLAS/0.3.9-GCC-9.3.0 CMake/3.16.4-GCCcore-9.3.0 Boost/1.71.0-gompi-2019b
```

Comment:
I think this is too specific to be useful for the broader community. I'd suggest using the bare minimum, and ignoring any amici dependencies here.

Reply:
it was meant to be usable with the `example_MPIPool.py`.

Reply:
I think a bare minimum example batch file would be preferable (which works e.g. for Rosenbrock) would be preferable. In addition specifying e.g. modules required for Bonna for AMICI could be added somewhere else -- e.g. the versions are likely to change here.

Reply:
> it was meant to be usable with the example_MPIPool.py.

That shouldn't require the last line. For the other lines, chances are pretty low that they will work on any other than Bonna.

Reply:
i removed the example slurm file and added the `mpiexec -np ...`-command to the docstring of MPIPool, i think that this command is the most essential part.